lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -301,6 +297,9 @@ func (i *InteractionHandlersImpl) ContainerGetStdoutHandler(params interaction.C
 			session.Stdout(),
 		),
 		params.ID,
+		func() {
+			session.Close()
```

Comment:
It's not clear to me what happens here if the connection with the personality is just dropped - we should not be triggering a close on the container side without ContainerCloseStdinHandler being explicitly called.